kit/src/elements/select/style.scss

```diff
@@ -1,12 +1,12 @@
 .select select{
     --webkit-appearance: none;
-    color: var(--text-color-bright);
+    color: var(--text-color-dark);
     appearance: none;
     border-radius: 5rem;
     min-width: fit-content;
     width: 100%;
     height: 40px;
-    background: url("data:image/svg+xml;utf8,<svg xmlns='http://www.w3.org/2000/svg' width='40' height='40' fill='white'><polygon points='0,0 40,0 20,20'/></svg>")
+    background: url("data:image/svg+xml;utf8,<svg xmlns='http://www.w3.org/2000/svg' width='40' height='40' fill='black'><polygon points='0,0 40,0 20,20'/></svg>")
```

Comment:
This is all included in the app, the entire svg is just the polygon. If you mean the `xmlns='http://www.w3.org/2000/svg'` I had to look that up because I didn't know what it was actually for (I copy pasted this svg code off stackoverflow.), it doesn't seem to be an actual link to something but instead a required string that exist on a svg. See more info here: https://stackoverflow.com/questions/18467982/are-svg-parameters-such-as-xmlns-and-version-needed